src/app/child-dev-project/attendance/add-day-attendance/add-day-attendance.component.html

```diff
@@ -20,6 +20,7 @@
     <app-roll-call
       *ngIf="currentStage === 1"
       [eventEntity]="event"
+      [sortParticipantsBy]="config?.sortParticipantsBy"
```

Comment:
Either this shouldn't be optional or the config itself should be

Reply:
sorry, I don't think I got what you mean.

To keep backward compatibility as well as the option of a simple setup, this view may not have any config at all - therefore `config?.`
How would you write this here differently with as few lines of code?

Reply:
Sorry that I haven't made this more clear.... In the component the config is declared as 
```ts
config: AddDayAttendanceConfig;
```
So what I meant was to either make it non-optional in the template, or make the config in the component itself optional. Seems like the second option is better then :D

Reply:
ah, got it. Done :+1: